vumi/persist/fake_redis.py

```diff
@@ -513,6 +514,21 @@ def persist(self, key):
             return 1
         return 0
 
+    # HyperLogLog operations
+
+    @maybe_async
+    def pfadd(self, key, *values):
+        hll = self._setdefault_key(key, HyperLogLog(0.01))
+        old_card = hll.card()
+        for value in values:
+            hll.add(value)
+        return hll.card() != old_card
+
+    @maybe_async
+    def pfcount(self, key):
+        hll = self._data.get(key, HyperLogLog(0.01))
+        return len(hll)
```

Comment:
`hll.card()` returns a float. `len(hll)` is equivalent to `round(hll.card())` and is what the usage section of hyperloglog's README uses.